DataBase/dataBase.py

```diff
@@ -1,166 +1,131 @@
 import psycopg2
 from configparser import ConfigParser
+from contextlib import contextmanager
```

Comment:
context manager in python provides a better way to allocate and release resources